format/spec.md

```diff
@@ -949,25 +977,25 @@ Schemas are serialized as a JSON object with the same fields as a struct in the
 
 Types are serialized according to this table:
 
-|Type|JSON representation|Example|
```

Comment:
Probably we should keep this with the same whitespace

Reply:
Yeah, no need to reformat this.

Reply:
Same explanation as above.

Reply:
It's fine for a new table, but for existing tables we don't want to make the changes any larger than necessary.